themes/default/content/registry/packages/kubernetes/how-to-guides/choosing-the-right-helm-resource-for-your-use-case.md

```diff
@@ -0,0 +1,101 @@
+
+### Recommendations by Use Case
+
+| Use Case | Recommended Resource |
+| --------- | ---------- |
+| [*Fire-and-forget* Helm Chart installation?](#fire-and-forget-helm-chart-installation) | [Helm Release]({{< relref "/registry/packages/kubernetes/api-docs/helm/v3/release">}}) |
+| [Interact with existing Helm-managed resources?](#interoperability-with-existing-helm-releases) | [Helm Release]({{< relref "/registry/packages/kubernetes/api-docs/helm/v3/release">}}) |
+| [Need to customize/modify Helm Charts through transformations?](#fine-grained-diffs-and-transformations) | [Helm Chart]({{< relref "/registry/packages/kubernetes/api-docs/helm/v3/chart">}}) |
+| [Need fine-grained diffs on Helm Chart updates?](#fine-grained-diffs-and-transformations) | [Helm Chart]({{< relref "/registry/packages/kubernetes/api-docs/helm/v3/chart">}}) |
+| [Enforce CrossGuard policies on all Kubernetes resources?](#enforcing-crossguard-policies-on-kubernetes-resources) | [Helm Chart]({{< relref "/registry/packages/kubernetes/api-docs/helm/v3/chart">}}) |
+
+#### *Fire-and-forget* Helm Chart Installation
+
+In many cases, users simply want to install an unmodified Chart and manage its configuration in their IaC tool of choice by specifying the chart and the values in code. While both the Chart and Release resources accomplish this with aplomb, we would recommend the use of Helm Release in such situations due to the broader support of Helm features such as Hooks.
+
+#### Interoperability with existing Helm releases
+
+If you have existing Helm Releases deployed through a version of the Helm CLI and wish to now integrate them in Pulumi, the `Helm Release` resource is your best choice. Currently component resources like `Chart` don't offer the ability to `import` existing resources.
+
+#### Fine Grained Diffs and Transformations
+
+`Chart` resources have direct access to the Kubernetes resources installed by the chart before installation. As a result, `Chart` resources support `transformations` which allow program authors to programmatically manipulate resources before they are installed by Pulumi. This is a very powerful tool which has enabled several advanced use cases for our users. Unfortunately, `Helm Release` does not have the same flexibility in offering transformations support.
+
+Similarly, `Chart` resources can enumerate underlying resources and their inputs, thus providing fine-grained diffs on individual resources and richer previews.
+
+If these are important to your environment or use case, then the `Helm Chart` resource should be preferred.
+
+#### Enforcing CrossGuard Policies on Kubernetes Resources
+
+`Chart` resources extract all Kubernetes objects and deploy them as Pulumi resources, making each of them privy to applicable CrossGuard policies. While resources installed by `Helm Release` are directly installed by Helm bypassing CrossGuard policies.
```

Comment:
```suggestion
`Chart` resources extract all Kubernetes objects and deploy them as Pulumi resources, allowing fine-grained policy enforcement on these resources with CrossGuard. Since Pulumi does not manage the underlying resources from Helm Release, you should choose `Chart` if you need to enforce policy on these resources.
```